src/components/Reviews.jsx

```diff
@@ -0,0 +1,35 @@
+import { getReviews } from "../api"
+import { useEffect } from "react"
+import { useState } from "react"
+
+const Reviews = () => {
+
+    const [reviews, setReviews] = useState([])
+
+    useEffect(() => {
+        getReviews()
+        .then((reviews) => {
+            setReviews(reviews)
+        })
+    }, [reviews])
```

Comment:
You don't need reviews in the dependency array, this should cause an infinite loop. You can leave this blank so it just loads all the reviews on first load